PyRDF/backend/Backend.py

```diff
@@ -54,6 +55,40 @@ def __init__(self, config={}):
         """
         self.config = config
 
+    @classmethod
```

Comment:
I would define this just as a class attribute before `__init__`:
```suggestion
    initialization = lambda: None
```
and add the documentation about the attribute to the docstring of class `Backend`. BTW I just realized that the `config` attribute is documented both in the class and for the instance, and it should be documented only in the instance (there is no class config).

Reply:
> I would define this just as a class attribute before __init__

I tried that, but then I got the following error:

```
TypeError: unbound method <lambda>() must be called with Backend instance as first argument (got nothing instead)
```

It could be a class attribute, but as soon as we call it, it will fail and would require us to run something link:

```
b = Backend()
b.initialization()
```

Reply:
`initialization = lambda cls:None` ?

Reply:
That does not work either, it tries to run:

```
Backend().initialization(cls)
```

which would end up passing the `cls` parameter to the user's `initialize` function.